src/trans.rs

```diff
@@ -446,8 +638,8 @@ impl<'v, 'tcx: 'v> BinaryenFnCtxt<'v, 'tcx> {
                 // TODO: works for refs to "our stack", but not the locals on the wasm stack yet
```

Comment:
I'm interpreting this TODO as meaning we don't have the analysis that spills any address-taken variables onto onto the Rust stack. Is that correct?